### DISCUSSION TITLE
npx为什么报错啊

### DISCUSSION THREAD

**Discussion by hulinhao233** [1715173329/UnblockNeteaseMusic#827] [Q&A] [answered]
---------------------------
Windows Script Host
---------------------------
脚本:	C:\Users\StarocZhang\node_modules\@unblockneteasemusic\server\precompiled\app.js
行:	1
字符:	12
错误:	语法错误
代码:	800A03EA
源: 	Microsoft JScript 编译错误

---------------------------
确定   
---------------------------

**Comment by 1715173329** [comment, marked as answer]
Windows 默认 js 解析器是内置 jscript 并非 nodejs，你可以更改文件关联，或克隆源码从命令行启动。
